README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
```

Comment:
`bootstrap` is currently the only place we alternate on the value of `npmClient`, and it seems cleaner that way (supporting multiple arguments properly without re-implementing yargs, for example)